instructure/canvas-ios
# Make file submission icon color reflect course color.

refs: MBL-14832
affects: Student
release note: Make file submission icon color reflect course color.

test plan:
- Submit a file upload.
- View the submission comments.
- The file type icon should be the course color.
